contracts/0.4.24/oracle/LidoOracle.sol

```diff
@@ -222,6 +267,31 @@ contract LidoOracle is ILidoOracle, AragonApp {
         _tryPush(_epochId);
     }
 
+    /**
+     * @notice To make oracles less dangerous,
+     *  we can bound rewards report by 0.1% increase in stake and 15% decrease in stake,
+     *  with both values configurable by DAO voting in case of extremely unusual circumstances.
+     *  daily_reward_rate_PPM = 1e6 * reward / totalPooledEther / days
+     **/
+    function reportSanityChecks(Report report) private {
+        (uint256 postTotalPooledEther,
+         uint256 preTotalPooledEther,
+         uint256 timeElapsed) = getLastCompletedReports();
+        // todo: use safeMath everywhere
+        if (postTotalPooledEther == 0 || timeElapsed == 0) return;  // it's possible at the beginning of the work with the contract or in tests
+        if (postTotalPooledEther >= preTotalPooledEther) {  // check profit constraint
+            uint256 reward = postTotalPooledEther - preTotalPooledEther;
+            uint256 allowedBeaconBalanceAnnualIncreasePPM = getAllowedBeaconBalanceAnnualRelativeIncrease() * postTotalPooledEther;
+            uint256 rewardAnnualizedPPM = 1e6 * reward * 365 days / timeElapsed;
```

Comment:
`356 days` is `1 years`, if I remember cirrectly.

Reply:
`1 years` is deprecated